winml/lib/Api.Image/inc/TensorToVideoFrameConverter.h

```diff
@@ -19,24 +19,31 @@ class TensorToVideoFrameConverter : public ImageConverter {
       _In_ winml::LearningModelSession& session,
       _In_ ID3D12Resource* input_tensor,
       _In_ const ImageTensorDescription& tensor_description,
-      _Inout_ wm::VideoFrame& destination_video_frame);
+      _Inout_ wm::VideoFrame& destination_video_frame
+  );
 
   // Function takes in a byte pointer to a CPUTensor
   // converts it to VideoFrame backed by either a SoftwareBitmap or D3DSurface,
   void SoftwareTensorToVideoFrame(
       _In_ winml::LearningModelSession& session,
       _In_ BYTE* CPU_tensor_to_convert,
       _In_ ImageTensorDescription tensor_description,
-      _Inout_ wm::VideoFrame& destination_video_frame);
+      _Inout_ wm::VideoFrame& destination_video_frame
+  );
 
   void ConvertBatchedDX12TensorToBuffers(
       _In_ ID3D12Resource* input_tensor,
       _In_ size_t buffer_size_in_bytes,
       _In_ _winml::D3DDeviceCache& device_cache,
-      _Inout_ const std::vector<wss::IBuffer>& buffers);
+      _Inout_ const std::vector<wss::IBuffer>& buffers
+  );
 
  private:
-  GUID _d3d11TextureGUID = {0x14bf1054, 0x6ce7, 0x4c00, {0xa1, 0x32, 0xb0, 0xf2, 0x11, 0x5D, 0xE0, 0x7f}};  // {14BF1054-6CE7-4C00-A132-B0F2115DE07F}
+  GUID _d3d11TextureGUID = {
+      0x14bf1054,
+      0x6ce7,
+      0x4c00,
+      {0xa1, 0x32, 0xb0, 0xf2, 0x11, 0x5D, 0xE0, 0x7f}};  // {14BF1054-6CE7-4C00-A132-B0F2115DE07F}
   GUID _handleGUID = {0x700148fc, 0xc0cb, 0x4a7e, {0xa7, 0xc0, 0xe7, 0x43, 0xc1, 0x9, 0x9d, 0x62}};
```

Comment:
Is there a setting to symmetrically place the first opening and last closing braces on separate lines (when the line doesn't fit)? Otherwise it's oddly inconsistent with other places. e.g. See below, where the opening braces *are* on separate lines:

```
  static ScenarioTestsApi api = {
      ScenarioCppWinrtTestsClassSetup,
      Sample1,
...
      BindMultipleCPUBuffersOutputsOnGpu,
  };
```

I mean, one wouldn't write the following :b...

```
  if (swprintf_s(full_name, L"%s[%u]", name, index) > 0) {
    object->SetName(full_name); }
```

Reply:
I think there's one for it too. Let me check...

Reply:
Done

Reply:
Thanks.